specifyweb/frontend/js_src/lib/components/Syncer/findNodePosition.ts

```diff
@@ -91,21 +107,18 @@ function findChildPosition(
  * Handles nested tags of same tagName (i.e, <cell> inside of <cell>)
  */
 const findClosing = (string: string): number | undefined =>
-  finder(string, f.true, undefined);
+  finder(string, undefined, f.true);
 
 /** Find the next nth instance of a direct child with a given tagName */
 function findChild(
   xml: string,
   tagName: string,
-  index: number
+  targetIndex: number
 ): number | undefined {
-  let currentIndex = 0;
+  let index = 0;
+  debugger;
```

Comment:
Looks like you forgot to remove this debugger statement

Reply:
good catch!
btw, i am often finding `debugger;` nicer than setting breakpoints in the dev tools. reasons:
- dev tools breakpoints may shift when you edit code - `debugger;` remain in place you want them in
- easier to write conditional breakpoints `if(someCondition) debugger;` - you can do conditional in dev tools, but you don't get typescript validation or an error message if you make a syntax error
  - if code was minified, the variable names in source map don't match the executed javascript - not an issue if your conditional breakpoint is defined in the code

but of course, you can then forget to remove the debugger statement - thus I could add an eslint rule for this